hwdb.d/70-mouse.hwdb

```diff
@@ -371,6 +371,10 @@ mouse:usb:v046dpc068:name:Logitech G500:
 mouse:usb:v046dpc332:name:Logitech Gaming Mouse G502:
  MOUSE_DPI=1200@1000 *2400@1000 3200@1000 6400@1000
 
+# Logitech G502 HERO SE
+mouse:usb:v046dpc08b:name:Logitech G502 HERO SE:
+ MOUSE_DPI=1200@1000 *2400@1000 3200@1000 6400@1000
```

Comment:
hmm, so the previous entry is about another Logitech Mouse with G502, though with a different vid/pid, but the measurements appear to be the same. Can you merge the two entries hence?

(but doesn't matter really)

Reply:
Sure, I had not noticed it was possible to merge two entries.
I just force pushed with the suggested change, hope it's better now.